modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm

```diff
@@ -294,6 +294,18 @@
 	name = "breathing tube implant"
 	item_type = /obj/item/organ/internal/cyberimp/mouth/breathing_tube
 
+// Personal Defense Weapons (For when the pharmacist must become the harmacist)
+
+/datum/armament_entry/company_import/deforest/defense
+	subcategory = "Personal Defense Equipment"
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+	interest_required = COMPANY_SOME_INTEREST
+
+/datum/armament_entry/company_import/deforest/defense/firefly
+	item_type = /obj/item/gun/ballistic/automatic/pistol/firefly
```

Comment:
this should probably be moved into the 'contraband' (aka, need to multitool the board) options,

Reply:
Done!